modules/database/sqlite/deactivate.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Actions to run when the module gets deactivated.
+ *
+ * @since n.e.x.t
+ * @package performance-lab
+ */
+
+/**
+ * Deletes the db.php file, and deactivates the module in the SQLite database.
+ *
+ * @since n.e.x.t
+ */
+return function() {
+	if ( ! file_exists( WP_CONTENT_DIR . '/db.php' ) ) {
+		return;
+	}
+
+	global $wp_filesystem;
+	if ( ! $wp_filesystem ) {
+		require_once ABSPATH . '/wp-admin/includes/file.php';
+		WP_Filesystem();
+	}
+	$wp_filesystem->delete( WP_CONTENT_DIR . '/db.php' );
```

Comment:
What, if anything, does Query Monitor do about handling its drop-in in this multi-server situation?  What about Redis? @johnbillion @tillkruss ?

Reply:
Object Cache Pro expects the user to place the drop-in during the build/deployment phase. 

It’s unrealistic to expect WordPress to handle that.

Reply:
> It’s unrealistic to expect WordPress to handle that.

Agree. I think we should update the drop-in to do nothing if it can't locate required files or the plugin is disabled instead of trying to delete the drop-in.